pkg/moq/moq_test.go

```diff
@@ -200,7 +200,7 @@ func TestMoqSkipEnsure(t *testing.T) {
 }
 
 func TestMoqWithResets(t *testing.T) {
```

Comment:
Since we have added a test package, we could use a golden test instead which would generate the output and save it as a golden file. Could you add a case under `TestMockGolden` and remove this test?

Reply:
great idea, i've gone ahead and used this approach, thanks.